SwanProjects/bin/swan_bash

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash -i
+#Author Omar.Zapata@cern.ch 2021
+
+clear
+if ! [ -x "$(command -v jq)" ]; then
+  echo 'Error: jq is not installed.' >&2
+  sleep 60
+  exit 1
+fi
+if [[ $# -gt 0 ]] ; then
+    PROJECT=$1
+    PROJECT_PATH="$HOME/SWAN_projects/$PROJECT"
+    PROJECT_FILE="$PROJECT_PATH/.swanproject"
+    
+    if [ -d "$PROJECT_PATH" ]
+    then
+      STACK=`jq '.stack' $PROJECT_FILE`
+      RELEASE=`jq '.release' $PROJECT_FILE`
+      PLATFORM=`jq '.platform' $PROJECT_FILE`
+      USER_SCRIPT="$PROJECT_PATH/.userscript"
+      env -i HOME=$HOME OAUTH2_FILE=$OAUTH2_FILE \
+             OAUTH2_TOKEN=$OAUTH2_TOKEN \
+             OAUTH_INSPECTION_ENDPOINT=$OAUTH_INSPECTION_ENDPOINT \
+             PROJECT=$PROJECT \
+             PROJECT_PATH=$PROJECT_PATH PS1="$PS1" \
+             PATH="/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/bin:/sbin:$HOME/.local/bin/" \
```

Comment:
Do you need to hardcode all these paths or you can just append `$HOME/.local/bin/` to the current `PATH`?

Reply:
for the isolation of the environment the script removes all the variables, even PATH,
I am setting this with the default paths defined in the system.
the variable $HOME/.local/bin/ is to work with the extension like user not like root when I am installing it in my home directory.

Reply:
> for the isolation of the environment the script removes all the variables, even PATH,

If you are running `env` it means you have some PATH set, doesn't it? I'm just saying you could take what is in PATH already (the default directories) instead of hardcoding them.

>  the variable $HOME/.local/bin/ is to work with the extension like user not like root when I am installing it in my home directory.

Sorry, I don't understand. What extension? SwanProjects? Why would it be installed in my home?

Reply:
* With env almost all environment varibales are removed including PATH and HOME, I am adding then using the default PATH defined in a clear os and I am adding other variables required by EOS.
* Yes, SwanProjects, if you want to develop it in your user account we have to set $HOME/.local/bin/, otherwise we have to install it like root to test it. do you prefer I remove it?

Reply:
> With env almost all environment varibales are removed including PATH and HOME, I am adding then using the default PATH defined in a clear os and I am adding other variables required by EOS.

Ok, anyway, this will not be necessary anymore once we don't have to isolate from the environment.

> Yes, SwanProjects, if you want to develop it in our user account we have to set $HOME/.local/bin/, otherwise we have to install it like root to test it. do you prefer I remove it?

So you mean this is just for testing? In production the extensions are not installed in HOME.

Reply:
removed PATH, I put a fixme comment in the code that will be removed when isolation is not needed anymore.